multirange.js

```diff
@@ -10,9 +10,18 @@ self.multirange = function(input) {
 		return;
 	}
 
-	var values = input.getAttribute("value").split(",");
-	var min = +input.min || 0;
-	var max = +input.max || 100;
+	var values = [];
+	if(input.hasAttribute("value")) {
+		values = input.getAttribute("value").split(",");
+	}
+	var min = 0;
+	var max = 100;
+	if(input.hasAttribute("min")) {
+		min = +input.min;
+	}
+	if(input.hasAttribute("max")) {
+		max = +input.max;
+	}
```

Comment:
Similarly, this could be rewritten as:

```js
var min = +(input.min || 0);
var max = +(input.max || 100);
```

The reason this should work, is that even if `min` is set to `0`, `i.min` would return `"0"`, which is truthy. If `min` is not set at all, it will return `""`, which is falsy.